news/2018-03-05-anime-remix-themed-spotlights-2018.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: "Anime Remix Themed Spotlights 2018"
+date: 2018-03-05 14:00:00 +0000
+---
+
+Greetings everyone! We are happy to announce that the Themed Spotlights are back! To celebrate this we prepared a special selection of anime remixes for you! Enjoy the songs while clicking those circles, hitting the drum, catching those healthy fruits and go octopus-mode with the squares!
```

Comment:
comma after this
the drum -> the drums
going octopus mode with the notes

Reply:
Fixed them, although technically in taiko you only use one drum. 🤔